main.py

```diff
@@ -86,6 +102,61 @@
 
             # TODO
 
+            if not message_author in element.members:
+                element.members.append(message_author)
+
+            message = creation_messages.cc_intro(member_ids)
+            
+            # Role objects (based on ID)
+            main_guild = client.get_channel(bot_spam).guild # Find the guild we're in
+            roles = main_guild.roles # Roles from the guild
+            game_master_role = discord.utils.find(lambda r: r.id == game_master, roles) #WUSDIS                     # \
+            dead_participant_role = discord.utils.find(lambda r: r.id == dead_participant, roles) #WUSDIS           # | Find various role objects
+            frozen_participant_role = discord.utils.find(lambda r: r.id == frozen_participant, roles) #WUSDIS       # /
+            default_permissions = {
+                main_guild.default_role: discord.PermissionOverwrite(read_messages=False),                     # \
+                frozen_participant_role: discord.PermissionOverwrite(send_messages=False),                     # |
+                dead_participant_role: discord.PermissionOverwrite(read_messages=True, send_messages=False),   # | Set the permissions for each role and the bot user
+                game_master_role: discord.PermissionOverwrite(read_messages=True),                             # |
+                self.bot.user: discord.PermissionOverwrite(read_messages=True),                                # /
+                **{                                                                                       # \
+                    member: discord.PermissionOverwrite(read_messages=True) for member in element.members # | Set permissions for each member
+                },                                                                                        # /
+            }
+
+            with open("conspiracy_channels/cc_data.json") as cc_data:
+                try:
+                    data = json.load(cc_data) # Try to load conspiracy channel data from a file, ./cc_data.json
+                except JSONDecodeError:
+                    return await client.get_channel(message.channel).send('cc_data was not found or is invalid.') # If we can't find it or it's invalid, throw an error
```

Comment:
Give the user more useful information. Tell them how they can get help.

Reply:
This is YOUR code! Are you now telling me how the code that you write should be written? 🤔

Reply:
This originally sent it to the console I believe, just change the message to say "ask a gm for help" or something